src/cluster_legacy.c

```diff
@@ -7026,32 +7032,46 @@ int clusterCommandSpecial(client *c) {
         } else {
             addReplyLongLong(c, clusterNodeFailureReportsCount(n));
         }
-    } else if (!strcasecmp(c->argv[1]->ptr, "failover") && (c->argc == 2 || c->argc == 3)) {
-        /* CLUSTER FAILOVER [FORCE|TAKEOVER] */
+    } else if (!strcasecmp(c->argv[1]->ptr, "failover") && (c->argc >= 2)) {
+        /* CLUSTER FAILOVER [FORCE|TAKEOVER] [replicaid <node id>] */
```

Comment:
```suggestion
        /* CLUSTER FAILOVER [FORCE|TAKEOVER] [REPLICAID <node id>] */
```

Reply:
If we want this to be documented, we should update the JSON file too: `src/commands/cluster-failover.json`.

Reply:
It should be internal. I am not sure whether we should document it.